FlowCrypt/Functionality/Services/Remote Pub Key Services/AttesterApi.swift

```diff
@@ -10,7 +10,7 @@ struct PubkeySearchResult {
 }
 
 protocol AttesterApiType {
-    func lookupEmail(email: String) async throws -> [KeyDetails]
+    func lookupEmail(_ email: String) async throws -> [KeyDetails]
```

Comment:
We can call it just `func lookup(email: String)` instead of `func lookupEmail(_ email: String)`, so it's more Swift-style and we don't need to use underscore.